qemu/tests/cfg/boot_with_max_mem.cfg

```diff
@@ -0,0 +1,7 @@
+- boot_with_max_mem:
+    virt_test_type = qemu
+    only Linux
+    type = boot_with_max_mem
+    start_vm = no
+    boot_mem = 8T
+    boot_smp = 2
```

Comment:
```
smp_fixed = 2                                                                                                     
vcpu_maxcpus=${smp_fixed} 
```

Reply:
To be more strictly,  add the topology:
```
vcpu_cores = 0                                                                  
vcpu_threads = 1                                                                
vcpu_sockets = 2                                                                                                                                                
vcpu_dies = 1 
```

Reply:
> ```
> smp_fixed = 2                                                                                                     
> vcpu_maxcpus=${smp_fixed} 
> ```
 I will update the code in the above way，thanks

Reply:
@yanan-fu, The update has been completed, please check it